docs/content/preview/yugabyte-platform/security/enable-encryption-in-transit.md

```diff
@@ -367,6 +367,50 @@ When you create a universe, you can enable TLS using certificates provided by Ha
 
 You can also edit TLS settings for an existing universe by navigating to **Universes**, opening a specific universe, clicking **Actions > Edit Security > Encryption in-Transit** to open the **TLS Configuration** dialog, and then modifying the required settings.
 
+## Kubernetes cert-manager
+
+For a universe created on Kubernetes, YugabyteDB Anywhere allows you to configure an existing running instance of the [cert-manager](https://cert-manager.io/) as a TLS certificate provider for a cluster, assuming that the following criteria are met:
+
+- The cert-manager is running in the Kubernetes cluster.
+- A root or intermediate CA (either self-signed or external) is already configured on the cert-manager. The same root certificate file must be prepared for upload to YugabyteDB Anywhere.
+- An Issuer or ClusterIssuer Kind of the cert-manager resource is prepared and is ready to issue certificates using the previously-mentioned root or intermediate certificate.
+
+During the universe creation, you can enable TLS certificates issued by the cert-manager, as follows:
+
+1. Upload the root certificate to YugabyteDB Anywhere:
+
+   - Prepare the root certificate in a file (for example, `root.crt`).
+   - Navigate to **Configs > Security > Encryption in Transit** and click **Add Certificate**.
+   - On the **Add Certificate** dialog shown in the following illustration, select **K8S cert-manager**:
+
+     ![img](/images/yp/security/kubernetes-cert-manager.png)
```

Comment:
This image has a typo in the 'test-k8s-cret' :). Can I share another screen shot to upload here?